src/components/Homepage/BodySection/Card/FeatureCard.tsx

```diff
@@ -2,8 +2,8 @@ import { CardProps } from '../../HomepageContent';
 import { Links } from '../../../ProductPage/BodySection/Card/Links';
 import { Image } from '../../../Image';
 
-export const FeatureCard = ({ title, content, image, links }: CardProps) => (
-  <div className="flex justify-between flex-col">
+export const FeatureCard = ({ title, content, image, links, link }: CardProps) => (
+  <a href={link} className="pointer flex justify-between flex-col">
```

Comment:
It's worth being mindful here of `validateDOMNesting` console errors that arise as a result of this ([reference](https://www.dhiwise.com/post/mastering-validatedomnesting-best-practices)). 

In short, it breaks the HTML standards if we put certain tags inside certain other tags (with `div`s, pretty much anything goes) - in this example, `a` tags from `<Links ... />` conflict with this outer `a` tag.

We should aim to reduce these to a minimum, so worth a look at restructuring this and testing the other pages here with this in mind.